src/convertToObject.js

```diff
@@ -1,18 +1,16 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes string with styles
- * and returns object where CSS properties are keys
- * and values are the values of related CSS properties
-
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedObj = {};
+
+  sourceString.split(';').map(stroke =>
+    stroke.split(':').map(element => element.trim()))
+    .filter(array => array.length > 1)
+    .forEach(array => {
+      convertedObj[array[0]] = array[1];
+    });
```

Comment:
Pay attention to indentations. Each chained method call should be on the same level.
```suggestion
  sourceString
    .split(';')
    .map(stroke =>
      stroke.split(':').map(element => element.trim()))
    .filter(array => array.length > 1)
    .forEach(array => {
      convertedObj[array[0]] = array[1];
    });
```

Reply:
got it, in future will no make these mistakes